airflow/upgrade/rules/fix_conf_not_importable_from_airflow.py

```diff
@@ -0,0 +1,29 @@
+
+from airflow.upgrade.rules.base_rule import BaseRule
+
+print(BaseRule)
+
+<<<<<<< HEAD
+
+class ConfImportFromAirflow(BaseRule):
+    """
+      ConfImportFromAirflow class to ensure proper import of conf from to work in Airflow 2.0
+      """
+    title = "Properly import conf from Airflow"
+    description = """\
+    In Airflow-2.0, it's not possible to import conf from airflow by using \'import conf from airflow\'
+    To ensure your code works in Airflow 2.0, you should use \'from airflow.Configuration import conf \'
+                      """
+
+    def check(self):
+        from airflow import conf
+        if conf:
+            return (
+                "Warning: Properly import conf to ensure your code works in AIrflow 2.0"
+                "From Airflow 2.0, you should use from airflow.Configuration import conf."
+            )
```

Comment:
What this rule should do is to scan all users DAGs and plugins and verify if they use `from airflow import conf` in their code. If yes, we should raise a warning for each file that is impacted by this issue.

Reply:
Got it. Thanks

Reply:
Hi Turbaszek, I have updated the patch. Kindly review. Thanks